solidity/dashboard/Dockerfile

```
@@ -3,6 +3,10 @@ FROM node:11.15.0-alpine as build
 
 WORKDIR /app
 
+ARG ETH_RPC_URL
+ARG CHAIN_ID
+ENV ETH_RPC_URL=$ETH_RPC_URL
+ENV CHAIN_ID=$CHAIN_ID
```

Maybe couple of updates for consistency with other scripts we've got?
```suggestion
ARG ETH_WS_URL
ARG ETH_NETWORK_ID
ENV ETH_WS_URL=$ETH_WS_URL
ENV ETH_NETWORK_ID=$ETH_NETWORK_ID
```